Content.Client/VoiceMask/VoiceMaskNameChangeWindow.xaml.cs

```diff
@@ -12,10 +12,27 @@ public sealed partial class VoiceMaskNameChangeWindow : FancyWindow
 {
     public Action<string>? OnNameChange;
     public Action<string?>? OnVerbChange;
+    public Action<string?>? OnSoundChange;
 
-    private List<(string, string)> _verbs = new();
+    /// <summary>
+    ///     List of all the loaded speech verbs (name, protoID).
+    /// </summary>
+    private List<(string, string)> _speechVerbs = new();
```

Comment:
Is there a reason the type is not (string, ProtoId<?>)?

Reply:
Unfortunately yes :(
It allows me make the `PopulateOptionButton` function usable for both! I couldn't figure out a way to make a generic protoID but if you know a way I'm all ears.